src/IJ_mv/IJMatrix_parcsr_device.c

```diff
@@ -11,10 +11,11 @@
  *
  *****************************************************************************/
 
+#include "_hypre_onedpl.hpp"
```

Comment:
Can you remind me again why this is not integrated into `_hypre_utilities.hpp`

Reply:
The oneDPL headers need to be included before any other C/C++ stl includes in order to work properly, and this was the cleanest way I found of doing this. Note that wherever `_hypre_onedpl.hpp` is included, it must always be the first include.

Reply:
Thanks!  Now I remember.